Makefile

```diff
@@ -0,0 +1,64 @@
+LOCAL_IMAGE = acorn_docker:1.0
+REMOTE_IMAGE = merlinran/acorn_docker
+
+.PHONY: list
+list:
```

Comment:
Since this is the first target in the file, `make` without a target simply runs this rule. And this rule has no dependencies.